webcompat/static/css/webcompat.dev.css

```diff
@@ -58,3 +58,7 @@
 .is-hidden {
   display: none;
 }
+
+.visually-hidden {
```

Comment:
I added this style because if we `display: none` the label/svg, the container collapses and it looks super broken.